test/e2e/apps/statefulset.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/onsi/ginkgo"
 	"github.com/onsi/gomega"
 	appsv1 "k8s.io/api/apps/v1"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```

Comment:
Looks good to me, but other reviewers would want this to be removed.

Reply:
Undid that. Appears to have change from my linter.